elasticsearch/elasticsearch.d.ts

```diff
@@ -331,6 +333,11 @@ declare module Elasticsearch {
         refresh?: boolean;
     }
 
+    export interface DeleteByQueryParams extends GenericParams {
+        index: string;
```

Comment:
Missing in Search:
[conflicts=abort] String — What to do when the delete-by-query hits version conflicts?

Alot of fields from the search api is missing in deletebyquery.
Can you point to some documentation that states they have the exacly the same api now by design and they will not differ. Since they already differ in the documentation ?

I get they are to be used in a similar way, but they are not the same IMO. 

Also please add a test for the method. It is quickly done. Just paste the example from the documentation into the test file.